scripts/image_constants.sh

```diff
@@ -9,7 +9,8 @@ SECURITY_GROUPS="default"
 TEST_INSTANCE_NAME="_test_$IMAGE_NAME"
 TEST_FLAVOR="standard.tiny"
 TEST_NET_ID="63b53b45-3e3f-454c-8a27-d1bfae422c11"
-TEST_FIP="86.50.170.139"
+TEST_FIP="86.50.170.201"
 TEST_PING_COUNT="5"
 TEST_PING_INTERVAL="20"
 PYTHONIOENCODING="utf-8"
+OS_DISTRO_PROPERTY="linux"
```

Comment:
Should this be a constant at all? How about setting it per image instead like IMAGE_NAME? Then it can be properly set for all images without workarounds.

Reply:
I think that before heading out to vacation I was simply trying to refrain from changing the other build scripts if possible. It would be the better approach for sure. Stay tuned.